<ui version="4.0" >
 <class>NetworksSettingsPage</class>
 <widget class="QWidget" name="NetworksSettingsPage" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>688</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" >
   <property name="leftMargin" >
    <number>0</number>
   </property>
   <property name="topMargin" >
    <number>0</number>
   </property>
   <property name="rightMargin" >
    <number>0</number>
   </property>
   <property name="bottomMargin" >
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QListWidget" name="networkList" >
         <property name="selectionBehavior" >
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="sortingEnabled" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" >
         <item>
          <widget class="QPushButton" name="renameNetwork" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text" >
            <string>Re&amp;name...</string>
           </property>
           <property name="icon" >
            <iconset resource="../../icons/icons.qrc" >:/16x16/actions/oxygen/16x16/actions/edit-rename.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="addNetwork" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text" >
            <string>&amp;Add...</string>
           </property>
           <property name="icon" >
            <iconset resource="../../icons/icons.qrc" >:/16x16/actions/oxygen/16x16/actions/list-add.png</iconset>
           </property>
           <property name="iconSize" >
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="deleteNetwork" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text" >
            <string>De&amp;lete</string>
           </property>
           <property name="icon" >
            <iconset resource="../../icons/icons.qrc" >:/16x16/actions/oxygen/16x16/actions/edit-delete.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="connectNow" >
           <property name="enabled" >
            <bool>false</bool>
           </property>
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text" >
            <string>Connect now</string>
           </property>
           <property name="icon" >
            <iconset/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="detailsBox" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
         <horstretch>10</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title" >
        <string>Network Details</string>
       </property>
       <layout class="QVBoxLayout" >
        <item>
         <layout class="QHBoxLayout" >
          <item>
           <widget class="QLabel" name="label_6" >
            <property name="text" >
             <string>Identity:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="identityList" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTabWidget" name="tabWidget" >
          <property name="enabled" >
           <bool>true</bool>
          </property>
          <property name="currentIndex" >
           <number>0</number>
          </property>
          <widget class="QWidget" name="serversTab" >
           <property name="enabled" >
            <bool>true</bool>
           </property>
           <attribute name="title" >
            <string>Servers</string>
           </attribute>
           <layout class="QVBoxLayout" >
            <item>
             <layout class="QVBoxLayout" >
              <item>
               <layout class="QHBoxLayout" >
                <item>
                 <widget class="QListWidget" name="serverList" >
                  <property name="sizePolicy" >
                   <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
                    <horstretch>1</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                 </widget>
                </item>
                <item>
                 <layout class="QVBoxLayout" >
                  <item>
                   <widget class="QPushButton" name="editServer" >
                    <property name="sizePolicy" >
                     <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="text" >
                     <string>&amp;Edit...</string>
                    </property>
                    <property name="icon" >
                     <iconset/>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="addServer" >
                    <property name="sizePolicy" >
                     <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="text" >
                     <string>&amp;Add...</string>
                    </property>
                    <property name="icon" >
                     <iconset resource="../../icons/icons.qrc" >:/16x16/actions/oxygen/16x16/actions/list-add.png</iconset>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="deleteServer" >
                    <property name="sizePolicy" >
                     <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="text" >
                     <string>De&amp;lete</string>
                    </property>
                    <property name="icon" >
                     <iconset resource="../../icons/icons.qrc" >:/16x16/actions/oxygen/16x16/actions/edit-delete.png</iconset>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" >
                    <property name="spacing" >
                     <number>6</number>
                    </property>
                    <property name="leftMargin" >
                     <number>0</number>
                    </property>
                    <property name="topMargin" >
                     <number>0</number>
                    </property>
                    <property name="rightMargin" >
                     <number>0</number>
                    </property>
                    <property name="bottomMargin" >
                     <number>0</number>
                    </property>
                    <item>
                     <spacer>
                      <property name="orientation" >
                       <enum>Qt::Horizontal</enum>
                      </property>
                      <property name="sizeHint" >
                       <size>
                        <width>0</width>
                        <height>20</height>
                       </size>
                      </property>
                     </spacer>
                    </item>
                    <item>
                     <widget class="QToolButton" name="upServer" >
                      <property name="toolTip" >
                       <string>Move upwards in list</string>
                      </property>
                      <property name="text" >
                       <string>...</string>
                      </property>
                      <property name="icon" >
                       <iconset resource="../../icons/icons.qrc" >:/16x16/actions/oxygen/16x16/actions/go-up.png</iconset>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QToolButton" name="downServer" >
                      <property name="toolTip" >
                       <string>Move downwards in list</string>
                      </property>
                      <property name="text" >
                       <string>...</string>
                      </property>
                      <property name="icon" >
                       <iconset resource="../../icons/icons.qrc" >:/16x16/actions/oxygen/16x16/actions/go-down.png</iconset>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <spacer>
                      <property name="orientation" >
                       <enum>Qt::Horizontal</enum>
                      </property>
                      <property name="sizeHint" >
                       <size>
                        <width>0</width>
                        <height>20</height>
                       </size>
                      </property>
                     </spacer>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <spacer>
                    <property name="orientation" >
                     <enum>Qt::Vertical</enum>
                    </property>
                    <property name="sizeHint" >
                     <size>
                      <width>20</width>
                      <height>40</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QCheckBox" name="randomServer" >
                <property name="enabled" >
                 <bool>true</bool>
                </property>
                <property name="text" >
                 <string>Choose random server for connecting</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="performTab" >
           <attribute name="title" >
            <string>Perform</string>
           </attribute>
           <layout class="QVBoxLayout" >
            <item>
             <widget class="QLabel" name="label" >
              <property name="text" >
               <string>Commands to execute on connect:</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" >
              <item>
               <widget class="QTextEdit" name="performEdit" >
                <property name="enabled" >
                 <bool>true</bool>
                </property>
                <property name="lineWrapMode" >
                 <enum>QTextEdit::NoWrap</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="autoIdentify" >
                <property name="enabled" >
                 <bool>true</bool>
                </property>
                <property name="title" >
                 <string>Auto Identify</string>
                </property>
                <property name="checkable" >
                 <bool>true</bool>
                </property>
                <property name="checked" >
                 <bool>true</bool>
                </property>
                <layout class="QVBoxLayout" >
                 <item>
                  <layout class="QGridLayout" >
                   <item row="0" column="0" >
                    <widget class="QLabel" name="label_2" >
                     <property name="text" >
                      <string>Service:</string>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="1" >
                    <widget class="QLineEdit" name="autoIdentifyService" >
                     <property name="enabled" >
                      <bool>true</bool>
                     </property>
                     <property name="text" >
                      <string>NickServ</string>
                     </property>
                    </widget>
                   </item>
                   <item row="1" column="0" >
                    <widget class="QLabel" name="label_3" >
                     <property name="enabled" >
                      <bool>true</bool>
                     </property>
                     <property name="text" >
                      <string>Password:</string>
                     </property>
                    </widget>
                   </item>
                   <item row="1" column="1" >
                    <widget class="QLineEdit" name="autoIdentifyPassword" >
                     <property name="enabled" >
                      <bool>true</bool>
                     </property>
                     <property name="echoMode" >
                      <enum>QLineEdit::Password</enum>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab" >
           <attribute name="title" >
            <string>Advanced</string>
           </attribute>
           <layout class="QVBoxLayout" >
            <item>
             <widget class="QGroupBox" name="groupBox_3" >
              <property name="enabled" >
               <bool>true</bool>
              </property>
              <property name="title" >
               <string>Network Encoding</string>
              </property>
              <layout class="QVBoxLayout" >
               <item>
                <layout class="QHBoxLayout" >
                 <item>
                  <layout class="QGridLayout" >
                   <item row="0" column="0" >
                    <widget class="QLabel" name="label_4" >
                     <property name="text" >
                      <string>Sending:</string>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="1" >
                    <widget class="QComboBox" name="sendEncoding" >
                     <property name="enabled" >
                      <bool>false</bool>
                     </property>
                     <property name="sizePolicy" >
                      <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="insertPolicy" >
                      <enum>QComboBox::InsertAlphabetically</enum>
                     </property>
                    </widget>
                   </item>
                   <item row="1" column="0" >
                    <widget class="QLabel" name="label_5" >
                     <property name="text" >
                      <string>Receiving:</string>
                     </property>
                    </widget>
                   </item>
                   <item row="1" column="1" >
                    <widget class="QComboBox" name="recvEncoding" >
                     <property name="enabled" >
                      <bool>false</bool>
                     </property>
                     <property name="sizePolicy" >
                      <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="insertPolicy" >
                      <enum>QComboBox::InsertAlphabetically</enum>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <spacer>
                   <property name="orientation" >
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" >
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QCheckBox" name="useDefaultEncodings" >
                 <property name="enabled" >
                  <bool>true</bool>
                 </property>
                 <property name="text" >
                  <string>Use defaults</string>
                 </property>
                 <property name="checked" >
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="autoReconnect" >
              <property name="enabled" >
               <bool>true</bool>
              </property>
              <property name="title" >
               <string>Automatic Reconnect</string>
              </property>
              <property name="checkable" >
               <bool>true</bool>
              </property>
              <layout class="QVBoxLayout" >
               <item>
                <layout class="QHBoxLayout" >
                 <item>
                  <layout class="QGridLayout" >
                   <item row="0" column="0" >
                    <widget class="QLabel" name="label_7" >
                     <property name="text" >
                      <string>Interval:</string>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="1" >
                    <widget class="QSpinBox" name="reconnectInterval" >
                     <property name="suffix" >
                      <string> s</string>
                     </property>
                     <property name="minimum" >
                      <number>10</number>
                     </property>
                     <property name="maximum" >
                      <number>3600</number>
                     </property>
                     <property name="value" >
                      <number>30</number>
                     </property>
                    </widget>
                   </item>
                   <item row="1" column="0" >
                    <widget class="QLabel" name="label_8" >
                     <property name="text" >
                      <string>Retries:</string>
                     </property>
                    </widget>
                   </item>
                   <item row="1" column="1" >
                    <widget class="QSpinBox" name="reconnectRetries" >
                     <property name="minimum" >
                      <number>1</number>
                     </property>
                     <property name="maximum" >
                      <number>999</number>
                     </property>
                     <property name="value" >
                      <number>10</number>
                     </property>
                    </widget>
                   </item>
                   <item row="1" column="2" >
                    <widget class="QCheckBox" name="unlimitedRetries" >
                     <property name="text" >
                      <string>Unlimited</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <spacer>
                   <property name="orientation" >
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" >
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QCheckBox" name="rejoinOnReconnect" >
                 <property name="text" >
                  <string>Rejoin all channels on reconnect</string>
                 </property>
                 <property name="checked" >
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" >
               <size>
                <width>386</width>
                <height>31</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../icons/icons.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>useDefaultEncodings</sender>
   <signal>toggled(bool)</signal>
   <receiver>sendEncoding</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>647</x>
     <y>238</y>
    </hint>
    <hint type="destinationlabel" >
     <x>471</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useDefaultEncodings</sender>
   <signal>toggled(bool)</signal>
   <receiver>recvEncoding</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>658</x>
     <y>238</y>
    </hint>
    <hint type="destinationlabel" >
     <x>471</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>unlimitedRetries</sender>
   <signal>toggled(bool)</signal>
   <receiver>reconnectRetries</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>443</x>
     <y>336</y>
    </hint>
    <hint type="destinationlabel" >
     <x>390</x>
     <y>338</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
